mesheryctl/internal/cli/root/root.go

```diff
@@ -110,7 +110,7 @@ func init() {
 		versionCmd,
 		system.SystemCmd,
 		perf.PerfCmd,
-		mesh.MeshCmd,
```

Comment:
Is this removing `mesheryctl mesh` and replacing it with `exp`? Ideally, any new command to be exposed under `exp` will retain its intended syntax structure; e.g. `mesheryctl exp mesh validate`.

Reply:
Hmm Okay

Reply:
No actually this was adding the entire mesh subcmd inside exp

Reply:
Nice. ✔️